### ISSUE TITLE
Do you plan to accept contribution in localization?

### ISSUE THREAD

**Issue by muhammadbahaa2001** [Lixkote/Startify#13]

**Comment by Lixkote** [comment]
Yeah, of course, i do accept **any** contribution, and much thanks for helping to shape this app 😀

**Comment by muhammadbahaa2001** [comment]
> Yeah, of course, i do accept **any** contribution, and much thanks for helping to shape this app 😀

Can you, provide/tell me where I can find translation files so I can start?

**Comment by Lixkote** [comment]
> > Yeah, of course, i do accept **any** contribution, and much thanks for helping to shape this app 😀
> 
> Can you, provide/tell me where I can find translation files so I can start?

The translation support is not ready yet, but i will inform you here once it will be ready.
